package.json

```diff
@@ -15,15 +15,12 @@
     "lint": "eslint . -c .eslintrc.js --ext .ts,.tsx",
     "openapi:bundle": "bundle-api-spec -i openapi/index.yaml.template -o openapi/index.yaml -V $npm_package_version",
     "pregenerate": "npm run openapi:bundle && rimraf ./generated",
-    "generate": "npm-run-all generate:backend:* generate:templates:* generate:definitions:* generate:client:*",
+    "generate": "npm-run-all generate:backend:* generate:definitions:* generate:client:*",
     "generate:definitions:internal": "rimraf ./generated/definitions/internal && shx mkdir -p ./generated/definitions/internal && gen-api-models --api-spec ./openapi/index.yaml --no-strict --out-dir ./generated/definitions/internal",
     "generate:backend:api-models": "shx rm -rf generated/backend && shx mkdir -p generated/backend && gen-api-models --strict 0 --api-spec https://raw.githubusercontent.com/pagopa/io-backend/v7.28.0/api_backend.yaml --out-dir generated/backend",
     "generate:backend:notification-models": "shx rm -rf generated/notifications && shx mkdir -p generated/notifications && gen-api-models --strict 0 --api-spec https://raw.githubusercontent.com/pagopa/io-backend/v7.16.0/api_notifications.yaml --out-dir generated/notifications",
     "generate:backend:notification-hub": "shx rm -rf generated/notifications && shx mkdir -p generated/notifications && gen-api-models --strict 0 --api-spec https://raw.githubusercontent.com/pagopa/io-backend/v7.16.0/notification_queue_messages.yaml --out-dir generated/notifications",
-    "generate:templates:mailvalidation": "shx rm -rf generated/templates/mailvalidation && shx mkdir -p generated/templates/mailvalidation && TEMPLATE_NAME='MailValidation' TEMPLATE_VERSION='v1.0.15' APPLIER_TEMPLATE_PATH='_scripts/emailApplierTemplates/mailValidation' TARGET_PATH='generated/templates/mailvalidation/index' ts-node _scripts/email_template_script.ts",
```

Comment:
Is `_script` folder still necessary?

Reply:
I believe that is not necessary anymore. I removed it in https://github.com/pagopa/io-functions-app/pull/311/commits/14171f8cf91396db7ce08c6c48953c7091adf8a0. thanks for the suggestion